src/main/webapp/mapScript.js

```diff
@@ -304,6 +277,7 @@ function deletePlace(placeId) {
     markersArray[actualIndex].marker.setMap(null);
     markersArray.splice(actualIndex, 1);
   }
+  drawRoute();
```

Comment:
Have you checked it? Maybe I do something wrong? ((( Because I tried to do that but it seems like in order to delete a marker from already drawn route we should remove the whole route from the map firstly( and only then redraw (right now it does nothing)

Reply:
Nono, it's my bad!